api/src/main/java/org/openmrs/api/impl/PersonServiceImpl.java

```diff
@@ -651,6 +619,21 @@ public RelationshipType saveRelationshipType(RelationshipType relationshipType)
 		return attrObjects;
 	}
 	
+	private String combineAttributes(String patientAttributeProperty, String userAttributeProperty, PERSON_TYPE personType) {
+		StringBuilder result = new StringBuilder();
```

Comment:
@Ankitkumar94 you can add `final` in front, to signal its only assigned once, here.